src/Python/requirements.txt

```diff
@@ -6,7 +6,7 @@ notebook==6.5.2
 numpy==1.22.0
 pytest==7.2.0
 pygments==2.12.0    # to avoid high-severity issue.
-pyzmq==25.0.0       # to workaround: https://github.com/zeromq/pyzmq/issues/1764
```

Comment:
This comment is no longer applicable since updating the version in https://github.com/microsoft/iqsharp/pull/771.